app/controllers/following_controller.rb

```diff
@@ -0,0 +1,45 @@
+class FollowingController < ApplicationController
+  before_action :require_login, :redirect_if_current_user_equals_params
```

Comment:
If you want to please code climate, in addition to `require_login` you could use `find_user` and then have @user set for you in the action.